libs/core/langchain_core/language_models/base.py

```diff
@@ -160,11 +161,12 @@ async def agenerate_prompt(
                 prompt and additional model provider-specific output.
         """
 
+    @deprecated("0.1.0", alternative="invoke")
     @abstractmethod
     def predict(
         self, text: str, *, stop: Optional[Sequence[str]] = None, **kwargs: Any
     ) -> str:
-        """Pass a single string input to the model and return a string prediction.
+        """[DEPRECATED] Pass a single string input to the model and return a string.
```

Comment:
No need to do that -- it's already done by the decorator

https://github.com/langchain-ai/langchain/blob/26f84b74d0f7dc4d2211a1a62d47eec36cb1d726/libs/core/langchain_core/_api/deprecation.py#L216-L216

Reply:
😮 ‼️